openedx/core/djangoapps/programs/rest_api/v1/tests/test_views.py

```diff
@@ -8,6 +8,7 @@
 from django.core.cache import cache
 from django.test.utils import override_settings
 from django.urls import reverse_lazy
+from common.djangoapps.student.models import CourseEnrollment
```

Comment:
I suspect the linter is going to complain about this -- it should be above line 14 I think?